transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py

```diff
@@ -220,114 +229,36 @@ def geocode_address(address):
     "PT_VC_VOV_PHYL",
     "PT_VC_VOV_ROBB",
     "PT_VC_VOV_SEXL",
-    # adding the new health and nutrition related indicators
```

Comment:
again here we should keep both health and poverty codes

Reply:
I believe they should come when we merge, no?

Reply:
Not sure! Looks like since the Health codes were merged into init.py after you branched then you don't have them at the init.py in the Poverty branch and I'm afraid if we merge now if will erase those Health related. Not sure how to handle this case in Git @jamescw =)